addons/account_check_printing/models/account_payment.py

```diff
@@ -20,7 +20,7 @@ def _compute_payment_method_line_id(self):
                 lambda l: l.payment_method_id == preferred
             )
             if record.payment_type == 'outbound' and method_line:
-                record.payment_method_line_id = method_line
+                record.payment_method_line_id = sorted(method_line, key=lambda x: x.sequence)[0]
```

Comment:
Is the `sorted` really needed? It is the default order
https://github.com/odoo/odoo/blob/3b21c328feee528e1b38b377371945365827562c/addons/account/models/account_payment_method.py#L84
So nothing should be done, unless I'm missing something

The only cases where it would be needed is if it something is done in the same transaction, where the cache is already filled, that would change the sequence... Not likely at all.

```suggestion
                record.payment_method_line_id = method_line[0]
```
Also, `Model.sorted()` would be preferred
```python
method_line.sorted()  # ensure the default order
method_line.sorted('sequence')  # short for sort on one field
method_line.sorted(lambda l: l.sequence)  # same as you but Odoo style (and a bit shorter)
```

Reply:
Wasn't needed indeed, thanks